gtsam/navigation/NavState.cpp

```diff
@@ -133,6 +388,30 @@ NavState NavState::retract(const Vector9& xi, //
 //------------------------------------------------------------------------------
 Vector9 NavState::localCoordinates(const NavState& g, //
     OptionalJacobian<9, 9> H1, OptionalJacobian<9, 9> H2) const {
+  // return LieGroup<NavState, 9>::localCoordinates(g, H1, H2);
```

Comment:
Why was retract unchanged but local coordinates changed? And, what’s the difference with example and logmap?

Reply:
@varunagrawal ?